test/Api/CatalogApiTest.php

```diff
@@ -211,50 +40,11 @@ public static function setUpBeforeClass() {
     }
 
     protected function setUp() {
-      $this->objects = [$this->beverages, $this->milks, $this->syrups, $this->coffee, $this->tea, $this->sales_tax];
-      $this->idMap = [];
-      $this->buildTestCatalog();
+      $this->catalogFixtures = new CatalogFixtures(self::$api);
     }
 
     protected function tearDown() {
-      $this->deleteTestCatalog();
-    }
-
-    protected function buildTestCatalog() {
-      $body = new \SquareConnect\Model\BatchUpsertCatalogObjectsRequest([
-        "idempotency_key" => uniqid(),
-        "batches" => [
-          [
-            "objects" => $this->objects
-          ]
-        ]
-      ]);
-
-      $response = self::$api->batchUpsertCatalogObjects($body);
-
-      foreach($response->getIdMappings() as $mapping) {
-        $this->idMap[$mapping->getClientObjectId()] = $mapping->getObjectId();
-      }
-    }
-
-    protected function deleteTestCatalog() {
-        $objectIds = [];
-        $cursor = "";
-        do {
-          $response = self::$api->listCatalog($cursor, null);
-          $cursor = $response->getCursor();
-          foreach($response->getObjects() as $object) {
-            array_push($objectIds, $object->getId());
-          }
-        } while (!empty($cursor));
-
-        while(!empty($objectIds)) {
-          $toRemove = array_splice($objectIds, 0, 200);
-          $request = new \SquareConnect\Model\BatchDeleteCatalogObjectsRequest([
-            "object_ids" => $toRemove
-          ]);
-          self::$api->batchDeleteCatalogObjects($request);
-        }
+      $this->catalogFixtures->cleanupTestCatalog();
```

Comment:
I considered that, but chose to explicitly clean up in the test teardown so that it is explicit when objects are deleted.